redi-connect-front/src/pages/front/signup/SignUpFormMentee.tsx

```diff
@@ -1,13 +1,13 @@
-import React from 'react';
-import { buildSignUpForm } from './signUpFormFactory';
-import { LoggedOutLayout } from '../../../layouts/LoggedOutLayout';
+import React from "react";
+import { buildSignUpForm } from "./signUpFormFactory";
+import Account from "../../../components/templates/Account";
 
-const Form = buildSignUpForm('public-sign-up-mentee-pending-review');
+const Form = buildSignUpForm("public-sign-up-mentee-pending-review");
 
 export default function SignUpFormMentee() {
   return (
-    <LoggedOutLayout>
+    <Account>
```

Comment:
Again, I don't really see how the name change (LoggedOutLayout => Account) makes sense. Particularly when I see it applied here.